packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -582,9 +596,9 @@ class PaginatedDataTableState extends State<PaginatedDataTable> {
                       data: const IconThemeData(
                         opacity: 0.54,
                       ),
-                      child: Ink(
```

Comment:
Why is the Ink removed?

Reply:
Not possible to find properties such as Ink.color in testing